shutdown_costmaps: false
controller_frequency: 25.0
planner_patience: 5.0
controller_patience: 5.0
planner_frequency: 1.0
oscillation_timeout: 10.0
oscillation_distance: 0.2
recovery_behaviors:
  - name: "conservative_reset"
    type: "clear_costmap_recovery/ClearCostmapRecovery"

  - name: "rotate_recovery"
    type: "rotate_recovery/RotateRecovery"

  - name: "aggressive_reset"
    type: "clear_costmap_recovery/ClearCostmapRecovery"

conservative_reset:
  reset_distance: 3.0
aggressive_reset:
  reset_distance: 0.0

layer_names: ["obstacle_layer"]
